satpy/etc/readers/fci_l1c_nc.yaml

```diff
@@ -1,11 +1,11 @@
 reader:
-  name: fci_l1c_fdhsi
-  short_name: FCI L1C FDHSI
-  long_name: MTG FCI Level 1C FDHSI
+  name: fci_l1c_nc
+  short_name: FCI L1C NC
+  long_name: MTG FCI Level 1C NetCDF
   description: >
-    Reader for FCI FDSHI data in NetCDF4 format.
+    Reader for FCI L1c data in NetCDF4 format.
```

Comment:
Should it be "1C" (lines 3 and 4) or "1c" (lines 6 and 8)?  What is the standard spelling for the letter in levels?  As it stands now, it's inconsistent.

Reply:
The consistency of this spelling should be checked in the rest of the PR as well.

Reply:
Good point. It's hard to find a standard as the EUM documentation is also not consistent :) But L1c seems the preferred way. I have updated the PR to use L1c consistently.